viewing_party/party.py

```diff
@@ -1,23 +1,148 @@
 # ------------- WAVE 1 --------------------
 
+from symbol import subscriptlist
+
+
 def create_movie(title, genre, rating):
-    pass
+  new_move = {"title": title,
+              "genre" : genre,
+              "rating": rating}
+  for key, value in new_move.items():
+    if value == None:
+      return None
+  else:
+      return new_move

```

Comment:
Nice work here! A couple of call outs:

Instead of first using the data to create a dictionary and then iterating over the keys/values to check if any of the values are none, we should first check if any of the arguments are invalid and immediately return None. 

Then we can avoid creating a `new_movie` and skip iterating. 

To do so in a Pythonic way, you can write:
```python 
if not title and not genre and rating: 
    return None

# then you can return a dictionary literal

return { "title": title,
              "genre" : genre,
              "rating": rating
          }
```

When you have an if statement that checks if the arguments are invalid and then return None, we're using the guard clause pattern. The Guard Clause pattern helps in these cases by “inverting” that way of thinking and rapidly returning false conditions and keeping the important code blocks (usually the longest) in the lowest indentation level possible. Read more [here](https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3)

This way, when another developer comes along, they can immediately recognize "Oh if these values aren't valid, then we'll return None."